docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -36,6 +36,20 @@ Remove the `#` at the beginning of the line to enable the setting.
 
 --
 
+[float]
+[[define-cluster__uuid]]
+==== Define `cluster_uuid`
+To bind the metrics of {ls} to a specifi cluster, define the `monitoring.cluster_uuid` 
+in configuration file (logstash.yml):
+
+
+[source,yaml]
+----------------------------------
+monitoring.cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
+----------------------------------
+
+--
+
```

Comment:
Other comments:
* Is this an optional or required step?  (If optional, let's notate that in the heading.)
* Is this a new step between our current steps 1 and 2?  If so, it should be listed as a step around line 18.  If it is a sub-section under "Disable default collection" then the heading needs one more `=` to format it correctly.

Reply:
Hi @karenzone thanks for watching. It's an optional step between 1 and 2. This is needed because on Kibana we want to group together the stats of LS and ES related to the same cluster. Metricbeat should bring this field and forward to ES